python/glow/functions.py

```diff
@@ -315,6 +315,36 @@ def normalize_variant(contigName: Union[Column, str], start: Union[Column, str],
     assert check_return_type(output)
     return output
 
+
+def impute_mean(array: Union[Column, str], missingValue: Union[Column, str] = None) -> Column:
```

Comment:
From a grammatical perspective, we impute the missing values, not the array or the mean(cf [the scipy docs](https://scikit-learn.org/stable/modules/impute.html)). So I think that a better name would be `impute_by_mean` or `impute_missing` or maybe just `impute` for shorthand. If we expect imputation by mean to be way more common than other methods (I think this is true?), I think it would make sense to have a string typed "strategy" argument that is `mean` by default.